app/celery/scheduled_tasks.py

```diff
@@ -53,6 +58,7 @@
 @statsd(namespace="tasks")
 def run_scheduled_jobs():
     try:
+        update_in_progress_jobs.apply_async(queue=QueueNames.JOBS)
```

Comment:
We really want this to run async? the updates won't take effect for the code that follows. I am not sure myself here, as I didn't dig deep into the process, but if following code depends the job latest updated values, it's possible it won't get it.

Reply:
Agreed, if the task to re-process the job runs before this task finishes updating the `updated_at` field of the job we'll end up in the same position. I'll change this to use `delay()` instead.